scripts/createFromTemplate.ps1

```diff
@@ -146,6 +146,32 @@ Copy-Item "$templateFolder/../scripts/torizonIO.ps1" "$location/.conf"
 Copy-Item "$templateFolder/../scripts/checkCIEnv.ps1" "$location/.conf"
 Copy-Item "$templateFolder/../scripts/validateDepsRunning.ps1" "$location/.conf"
 
+# Check if there are scripts defined in the .conf/deps.json of the template and, if so,
+# copy them to the .conf of the project
+$_deps = Get-Content "$templateFolder/.conf/deps.json" | ConvertFrom-Json
+
+# If there are installation scripts listed on the .conf/deps.json of the template
+if (($_deps.installDepsScripts.Count -gt 0)) {
+    # Create the .conf/install-deps-scripts if it doesn't exist and there are
+    # installation scripts that have the path .conf/install-deps-scripts
+    if (-not (Test-Path -Path "$location/.conf/install-deps-scripts" )){
```

Comment:
Let's maintain consistency and always use camel case if need more than one word in files ors `installDepsScripts`

Reply:
Sure, I will change it here